.travis.yml

```diff
@@ -1,19 +1,18 @@
 language: php
 
 php:
-  - 5.6
-  - 7.0
-  - 7.1
+  - "5.6"
```

Comment:
Why were the double quotes added? Seemed to work without them before.

Reply:
yes it worked without them, but was just following how it was mentioned in the docs. It expects a string rather than a number.

Reply:
Also, I think based on this info, it seems that the number is needed for more precise version such as 7.1.9. Just an assumption. 
[https://docs.travis-ci.com/user/languages/php/#choosing-php-versions-to-test-against](url)

Reply:
Both versions exist in the docs but let's use it with double quotes since I don't have a strong opinion on this and the docs don't state anything specific.